src/main/java/edu/ie3/datamodel/io/sink/InfluxDbSink.java

```diff
@@ -146,8 +146,8 @@ private Point transformToPoint(ResultEntity entity, String measurementName)
    *
    * @param timeSeries the time series to transform
    */
-  private <E extends TimeSeriesEntry<V>, V extends Value> Set<Point> transformToPoints(
-      TimeSeries<E, V> timeSeries) throws ProcessorProviderException {
+  private <E extends TimeSeriesEntry<? extends Value>, V extends Value>
```

Comment:
just here but generally is my question if we can avoid the changes to any type in this PR?

Reply:
I will adapt this and all the other parts.